### PR TITLE
DBRE-3992: use bbc-registry image for kafka-prometheus-jmx-exporter

### PR DESCRIPTION
**Pull request by aymeric-ledizes**

### CODE REVIEW

File: charts/cp-control-center/values.yaml [hunk 1]
```diff
@@ -78,8 +78,8 @@ prometheus:
   ## ref: https://github.com/prometheus/jmx_exporter
   jmx:
     enabled: true
-    image: solsson/kafka-prometheus-jmx-exporter@sha256
-    imageTag: 6f82e2b0464f50da8104acd7363fb9b995001ddff77d248379f8788e78946143
+    image: eu.gcr.io/bbc-registry/kafka-prometheus-jmx-exporter
+    imageTag: 32d09291efb2b25aee6b5f694506196fed5f1365946af5884ce86a0e2039915e
```
Comment (frivoire) [inline charts/cp-control-center/values.yaml:82]: Using the hash is not our best-practice.
You should modify the Dockerfile (cf: https://github.com/blablacar/dockerfiles/pull/2391#pullrequestreview-2148352845) to have a version

File: charts/cp-control-center/values.yaml [hunk 2]
```diff
@@ -78,8 +78,8 @@ prometheus:
   ## ref: https://github.com/prometheus/jmx_exporter
```
Comment (frivoire) [inline charts/cp-control-center/values.yaml:78]: Global remark: this PR is modifying a lot of charts, but I'm really not sure we are using all of them.
So, you should probably simplify and only modify the one that are useful :)

**Review by frivoire — CHANGES_REQUESTED** [review]

File: charts/cp-control-center/values.yaml (hunk 1 above)
Comment (aymeric-ledizes) [reply charts/cp-control-center/values.yaml:82]: The thing here is that there is not tag on the upstream version so I don't like to add semver for something we don't own niethr change anything expect doing a simple copy in our registry ?

File: charts/cp-control-center/values.yaml (hunk 2 above)
Comment (aymeric-ledizes) [reply charts/cp-control-center/values.yaml]: Ok done 👍

**Review by frivoire — COMMENTED** [review]
Changing my review to "neutral", to not block you from merging if you disagree with my last remark (above)

**Pull request merged**